### PR TITLE
fix adapter startup and add test

### PR DESCRIPTION
**Pull request by alexgraham-da**

### CODE REVIEW

File: exberry_adapter/test/daml/Tests/Matching.daml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+module Tests.Matching where
```
Comment (alexgraham-da) [inline exberry_adapter/test/daml/Tests/Matching.daml:1]: I believe there used to be an `integrationTesting` model in this repo (I'm not sure when or why it was removed). This is similar, but runs the full test automatically. The test is basic but could be extended, possibly for a larger volume stress test.

File: daml.yaml [hunk 2]
```diff
@@ -5,6 +5,8 @@ init-script: Demo:demo
 parties:
   - Alice
   - Bob
+  - Dana
+  - Carl
```
Comment (alexmatson-da) [inline daml.yaml:9]: Do we use these parties in our tests? They are newly introduced in this PR, and the only other place I see them referenced is in `getExistingParties`. Can they be dropped?

Comment (alexgraham-da) [reply daml.yaml:9]: They are used elsewhere in `Tests.Clearing.Setup` and I think `Distribution.Auction` (although I see now it's `Charlie` not `Carl`, so I will change). They aren't used in every test, but they are allocated in all tests via `allocateParties`. Not sure if it really helps or hurts having them all here, I believe the only thing it does is populate the navigator dropdown.

Comment (alexmatson-da) [reply daml.yaml]: Yep makes sense, it doesn't hurt to have them if they are used in tests

**Review by alexmatson-da — APPROVED** [review]

**Review by Sarahb-da — APPROVED** [review]

File: exberry_adapter/test/daml/Tests/Matching.daml (hunk 1 above)
Comment (brian-weir) [reply exberry_adapter/test/daml/Tests/Matching.daml:1]: It was removed as the code of the `integrationTesting` went out of sync very quickly and it was a very manual task to execute the testing in that you had to modify the code for almost each test followed by calling the various functions available in the correct order. The purpose of the `integrationTesting` was to test our daml models with exberry before the UI was finished.

**Review by brian-weir — APPROVED** [review]

**Pull request merged**